src/main/scala/ProcessOSM.scala

```diff
@@ -629,6 +633,122 @@ object ProcessOSM {
     import geoms.sparkSession.implicits._
 
     geoms
-      .join(changesets.select('id as 'changeset, 'uid, 'user), Array("changeset"))
+      .join(changesets.select('id as 'changeset, 'uid, 'user), Seq("changeset"))
   }
+
```

Comment:
Stuff from here down can probably be dropped, as it's more utility than core.

Reply:
Some of that stuff is useful.  I was thinking of moving it to a separate location.